grammar.rustpeg

```diff
@@ -604,6 +604,9 @@ type_qualifier0 -> TypeQualifier =
     K<"const"    / gnu<"__const">> { TypeQualifier::Const } /
     K<"restrict" / gnu<"__restrict" "__"?>> { TypeQualifier::Restrict } /
     K<"volatile" / gnu<"__volatile" "__"?>> { TypeQualifier::Volatile } /
+    clang<"_Nonnull"> { TypeQualifier::Nonnull } /
+    clang<"_Null_unspecified"> { TypeQualifier::NullUnspecified } /
+    clang<"_Nullable"> { TypeQualifier::Nullable } /
```

Comment:
Literal keywords need to be wrapped with `K<>` macro to make sure they are terminated by a non-keyword character, like this: `clang<K<"_Nonnull">>`